src/lib/support/crypto/EllipticCurve.h

```diff
@@ -66,6 +66,11 @@ using nl::Weave::ASN1::OID;
 class EncodedECPublicKey
 {
 public:
+    enum
+    {
+        kMaxValueLength = 2 * ((WEAVE_CONFIG_MAX_EC_BITS + 7) / 8) + 1
```

Comment:
Use "round up" or "align" macro to more clearly indicate what's being done here.

Reply:
Updated to use RoundUp() function.